streaming/base/format/base/writer.py

```diff
@@ -94,6 +94,13 @@ def __init__(self,
                 raise ValueError(f'`size_limit` must be greater than zero, instead, ' +
                                  f'found as {size_limit_value}.')
 
+        # Validate keyword arguments
+        invalid_kwargs = [
+            args for args in kwargs.keys() if args not in ('progress_bar', 'max_workers')
```

Comment:
`args` -> `arg`

why kwargs.keys()? isn't .keys() pointless here

Reply:
updated. Kept `keys()` just to be more specific. No harm either ways.